qcodes/instrument/channel.py

```diff
@@ -188,6 +192,7 @@ def __init__(self, parent: InstrumentBase,
         self._chan_type = chan_type
         self._snapshotable = snapshotable
         self._paramclass = multichan_paramclass
+        self.index_origin = index_origin
```

Comment:
shall this attribute be privte so that it's not user-adjustible after init?
```suggestion
        self._index_origin = index_origin
```

Reply:
Right, it doesn't make sense to change the index origin after the initialization.